scripts/rff/dc_generated/report.json

```diff
@@ -0,0 +1,989 @@
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Empty value found :: value: '', column: 'Kurtosis_DailyMaxTemperature', property: 'value', node: 'E:WeatherVariability_Counties->E9'",
+    "counterKey": "StrSplit_EmptyToken_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Found a missing or empty property value :: property: 'value', node: 'E:WeatherVariability_Counties->E9', type: 'StatVarObservation'",
+    "counterKey": "Sanity_MissingOrEmpty_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Empty value found :: value: '', column: 'Skewness_DailyMaxTemperature', property: 'value', node: 'E:WeatherVariability_Counties->E8'",
+    "counterKey": "StrSplit_EmptyToken_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Found a missing or empty property value :: property: 'value', node: 'E:WeatherVariability_Counties->E8', type: 'StatVarObservation'",
+    "counterKey": "Sanity_MissingOrEmpty_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Empty value found :: value: '', column: 'StandardDeviation_DailyMaxTemperature', property: 'value', node: 'E:WeatherVariability_Counties->E7'",
+    "counterKey": "StrSplit_EmptyToken_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Found a missing or empty property value :: property: 'value', node: 'E:WeatherVariability_Counties->E7', type: 'StatVarObservation'",
+    "counterKey": "Sanity_MissingOrEmpty_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Empty value found :: value: '', column: 'Kurtosis_DailyMinTemperature', property: 'value', node: 'E:WeatherVariability_Counties->E6'",
+    "counterKey": "StrSplit_EmptyToken_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "6"
+    },
+    "userMessage": "Found a missing or empty property value :: property: 'value', node: 'E:WeatherVariability_Counties->E6', type: 'StatVarObservation'",
+    "counterKey": "Sanity_MissingOrEmpty_value"
+  }, {
+    "level": "LEVEL_WARNING",
+    "location": {
+      "file": "WeatherVariability_Counties.csv",
+      "lineNumber": "2"
+    },
+    "userMessage": "Failed reference existence check :: value-ref: 'GeoId', property: 'variableMeasured', node: 'E:WeatherVariability_Counties->E0'",
```

Comment:
This suggests that there was a `variableMeasured: dcs:GeoId`.  Could it be from a previous version of TMCF?

Reply:
This is explained by the TMCF node, so I'll resolve this.